.ci/validation.sh

```diff
@@ -1001,6 +1001,17 @@ no-warning-imports-java-design-patterns)
   fi
   ;;
 
+git-diff)
+  if [ "$(git status | grep 'Changes not staged\|Untracked files')" ]; then
+    printf "Please clean up or update .gitattributes file.\nGit status output:\n"
+    git status
+    printf "Top 300 lines of diff:\n"
+    git diff | head -n 300
+    sleep 5s
```

Comment:
please remove this line it isnot required.
it was rewquieed only in Travis.

Reply:
Changes Done

Reply:
No done.
I still see it